docs/content/configuration.md

```diff
@@ -548,7 +548,7 @@ The following signing algorithms are supported:
 
 ### Caching
 
-Caching represents the configuration of the inter-query cache that built-in functions can utilize.
+Caching represents the configuration of the inter-query cache that built-in functions can utilize. Note: There is a [known issue](https://github.com/open-policy-agent/opa/issues/3042) with how the cache calculates memory usage, resulting in more memory usage than expected.
```

Comment:
Can we just remove this as we are going to address the issue shortly.

Reply:
We can if you insist, but I kept it on the off chance that it saves someone days of hair pulling (I wish I had seen that note 😄 ).